From 463f3f85e326253feb35015a022ab6d1e03fe1d5 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Aug 2022 20:50:27 -0700
Subject: [PATCH] Remove duplicate include of <sys/mount.h>

<sys/mount.h> is already included by lapi/mount.h.

This fixes build with glibc 2.36+

Upstream-Status: Submitted [https://github.com/linux-test-project/ltp/pull/953]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 testcases/kernel/syscalls/statx/statx09.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/testcases/kernel/syscalls/statx/statx09.c b/testcases/kernel/syscalls/statx/statx09.c
index aea329e086..34c5f41f87 100644
--- a/testcases/kernel/syscalls/statx/statx09.c
+++ b/testcases/kernel/syscalls/statx/statx09.c
@@ -18,7 +18,6 @@
  */
 
 #define _GNU_SOURCE
-#include <sys/mount.h>
 #include <stdlib.h>
 #include "tst_test.h"
 #include "lapi/fs.h"
-- 
2.37.1

